infra/dev/setup-env.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# build the container with the same UID and GID as the local user and launch the container
+USER_ID=$(id -u) GROUP_ID=$(id -g)  docker-compose up --build
```

Comment:
What is the purpose of this `USER_ID=$(id -u) GROUP_ID=$(id -g)`? can't it be placed in the Dockerfile?

Reply:
no, we need to run these command in the terminal, so I made a script for a simple use

Reply:
This is a shell command, must be run from terminal.